api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/R3DiagnosisTranslatorImpl.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.fhir2.api.translators.impl;
+
+import javax.annotation.Nonnull;
+
+import org.hl7.fhir.r4.model.Condition;
+import org.openmrs.Diagnosis;
+import org.openmrs.module.fhir2.api.annotations.R3Provider;
+import org.springframework.stereotype.Component;
+
+@Component("r3DiagnosisTranslator")
+@R3Provider
```

Comment:
This annotation is meant solely for providers.

Reply:
We can add a new one, but I don't think we need an R3 specific translation here. If there are errors when translating due to the clinical status, just handle this transform in the R3 provider.